engines/rdma.c

```diff
@@ -1029,19 +1053,37 @@ static int fio_rdmaio_close_file(struct thread_data *td, struct fio_file *f)
 	return 0;
 }
 
-static int aton(struct thread_data *td, const char *host,
-		     struct sockaddr_in *addr)
+static int fill_sockaddr(struct thread_data *td, const char *host,
+			 struct sockaddr_storage *sin, int af, unsigned short port)
```

Comment:
Looks like the style in this file for multi-lines is indenting with the bracket - https://github.com/axboe/fio/blob/ba8fe8672c82e2ebaa7f5ccb74d079933cc20eed/engines/rdma.c#L237 so would you be able to rewrap this?

Reply:
I'm not sure I understand this peace, can you please elaborate.

Reply:
Oops, I missed this. For long lines that get split we normally line up at the bracket of the previous line so I think it would be better as:
```
static int fill_sockaddr(struct thread_data *td, const char *host, struct
                         sockaddr_storage *sin, int af, unsigned short port)
```

Notice where the s of sockaddr_storage is compared to the bracket of the previous line.